### PR TITLE
Added audience feedback handling to Portal

### PR DESCRIPTION
**Pull request by SimonBackx**
fixes https://github.com/TryGhost/Team/issues/2048

Changes so Portal can handle feedback links and use the members API to store a vote.

### CODE REVIEW

File: ghost/portal/src/components/pages/FeedbackPage.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            <AccountHeader />
+            <h1 className="gh-portal-main-title">Saving failed.</h1>
+            <div>
+                <p className="gh-portal-text-center">{error}</p>
+            </div>
+        </div>
+    );
+}
+
+export default function UnsubscribePage() {
```
Comment (lenabaidakova) [inline ghost/portal/src/components/pages/FeedbackPage.js:65]: `FeedbackPage`? :)

**Review by lenabaidakova — APPROVED** [review]
🚀

File: ghost/portal/src/components/pages/FeedbackPage.js (hunk 1 above)
Comment (SimonBackx) [reply ghost/portal/src/components/pages/FeedbackPage.js:65]: Good catch!

Comment (SimonBackx) [reply ghost/portal/src/components/pages/FeedbackPage.js:65]:
```suggestion
export default function FeedbackPage() {
```

**Pull request merged**